### PR TITLE
Update document for HPA

### PR DESCRIPTION
**Pull request by jmesnil**

### CODE REVIEW

File: doc/apis.adoc [hunk 1]
```diff
@@ -44,7 +44,8 @@ It uses a `StatefulSet` with a pod spec that mounts the volume specified by `sto
 | `bootableJar` | BootableJar specifies whether the application image is using WildFly S2I Builder/Runtime images or Bootable Jar. If omitted,
 it defaults to false (application image is expected to use WildFly S2I Builder/Runtime images) | bool | false
 | `standaloneConfigMap` | spec to specify how standalone configuration can be read from a `ConfigMap` | *<<standaloneconfigmapspec>> |false
-| `resources`| Resources spec to specify the request or limits of the Stateful Set. If ommited, the namespace defaults are used. | *<<resources>> | false
+| `resources`| Resources spec to specify the request or limits of the Stateful Set. If ommited, the namespace defaults are used (https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/[more info]). | *<<Resource
+s>> | false
```
Comment (yersan) [inline doc/apis.adoc:48]: There is a line break in the middle of "Resources"

File: doc/apis.adoc [hunk 2]
```diff
@@ -44,7 +44,8 @@ It uses a `StatefulSet` with a pod spec that mounts the volume specified by `sto
 | `bootableJar` | BootableJar specifies whether the application image is using WildFly S2I Builder/Runtime images or Bootable Jar. If omitted,
 it defaults to false (application image is expected to use WildFly S2I Builder/Runtime images) | bool | false
 | `standaloneConfigMap` | spec to specify how standalone configuration can be read from a `ConfigMap` | *<<standaloneconfigmapspec>> |false
-| `resources`| Resources spec to specify the request or limits of the Stateful Set. If ommited, the namespace defaults are used. | *<<resources>> | false
+| `resources`| Resources spec to specify the request or limits of the Stateful Set. If ommited, the namespace defaults are used (https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/[more info]). | *<<Resource
```
Comment (yersan) [inline doc/apis.adoc:47]: s/If ommited/If omitted

File: doc/apis.adoc (hunk 1 above)
Comment (jmesnil) [reply doc/apis.adoc]: fixed, thanks

File: doc/apis.adoc (hunk 2 above)
Comment (jmesnil) [reply doc/apis.adoc]: fixed, thanks

**Review by yersan — APPROVED** [review]

**Pull request merged**